.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: File a bug report
+title: "[BUG] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **PLEASE READ THIS FIRST:**
+        - DO NOT use the bug and feature tracker for general questions and support requests.
+        Use the `cython-users` mailing list instead.
+        It has a wider audience, so you get more and better answers.
+        - Did you search for SIMILAR ISSUES already?
+        Please do, it helps to save us precious time that we otherwise could not invest into development.
+        - Did you try the LATEST MASTER BRANCH or pre-release?
+        It might already have what you want to report.
+        Specifically, the legacy stable 0.29.x release series receives only important low-risk bug fixes.
+        Also see the Changelog regarding recent changes:
+        https://github.com/cython/cython/blob/master/CHANGES.rst
+  - type: textarea
+    id: describe
+    attributes:
+      label: "Describe the bug"
+      description: "A clear and concise description of what the bug is."
+      placeholder: "Tell us what you see!"
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: "Code to reproduce the behaviour:"
+      value: |
+        ```cython
+        ...
```

Comment:
```suggestion
        # broken code
```

Reply:
I don't really like "broken code" because it implies that the code is broken rather than Cython is broken!

Reply:
It would be better to change it to something else, not necessarily `broken code`
![image](https://user-images.githubusercontent.com/52697657/183974637-fec7f3e0-bd12-4a10-9359-4c51fe0e948a.png)